mypy/nodes.py

```diff
@@ -390,6 +405,10 @@ def set_line(self, target: Union[Token, Node, int]) -> Node:
             self.initialization_statement.set_line(self.line)
             self.initialization_statement.lvalues[0].set_line(self.line)
 
+    def set_column(self, target: Union[Token, Node, int]) -> Node:
+        super().set_column(target)
+        return self
```

Comment:
inconsistent to return `self`? do we need to do the same stuff with `self.initializer`, `self.variable`, and `self.initialization_statement` as above in `set_line`?